auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -82,8 +82,8 @@
                       <strong>Alert:</strong><br>
                       <span>
                         The Future Effective Date for this filing is in
-                        {{ daysLeft(item.effectiveDate) }}
-                        {{ daysLeft(item.effectiveDate) === 1 ? 'day' : 'days' }}.
+                        {{ daysLeft(item.futureEffectiveDate) }}
+                        {{ daysLeft(item.futureEffectiveDate) === 1 ? 'day' : 'days' }}.
```

Comment:
Similar comment to above. A new computed could return "x days".

Reply:
Updated, using daysLeftText()